src/hooks/use-is-valid-wp-cli-inline.ts

```diff
@@ -0,0 +1,32 @@
+import { parse } from 'shell-quote';
+
+const PLACEHOLDER_CHAR_BEGIN = [ '<', '[', '{', '(' ];
+
+export function useIsValidWpCliInline( command: string ) {
+	const wpCliArgs = parse( command )
+		.map( ( arg ) => {
+			if ( typeof arg === 'string' || arg instanceof String ) {
+				return arg;
+			} else if ( 'op' in arg ) {
+				return arg.op;
+			} else {
+				return false;
+			}
+		} )
+		.filter( Boolean ) as string[];
+	const wpCommandCount = wpCliArgs.filter( ( arg ) => arg === 'wp' ).length;
+	const containsPath = wpCliArgs.some(
+		( arg ) => /path/i.test( arg[ 0 ] ) || /\//.test( arg[ 0 ] )
+	);
```

Comment:
> `/path/i.test( arg[ 0 ] )`

I presume we want to check the regular expression with the full string.

> `/\//.test( arg[ 0 ] )`

If we simply want to check if the string starts with `/` character, we can use the function `startsWith` instead of a regular expression.

```suggestion
	const containsPath = wpCliArgs.some( ( arg ) => /path/i.test( arg ) || arg.startsWith( '/' ) );
```

Reply:
This will cover some cases but I want to note that this won't work on Windows due to the path format (Windows vs. POSIX). In any case, I know it's definitely tricky to identify when an argument is path 😞 .

Reply:
Great point. Changed here! https://github.com/Automattic/studio/pull/286/commits/66b8730c23f5558ec5e9cbf8021b8cc8c9664dee

I was tempted to add the character `/` to the `PLACEHOLDER_CHAR_BEGIN` array. But in the end I preferred to keep that separate.